### DISCUSSION TITLE
Mac & ugs-platform-app-ios-4.dmg

### DISCUSSION THREAD

**Discussion by MikeAghevli** [winder/Universal-G-Code-Sender#1823] [Help]
Hi ! 
old man needs help "hobby" using cnc1419!
I have Mac and try to install ugs-platform-app-ios-4.dmg nothings happen 
Do I need some other pgm??
needed badly UGS

**Comment by breiler** [comment]
Which version did you download?

It is difficult to help with the description "nothing happens". Which steps are you taking to try and install the software?

**Comment by MikeAghevli** [comment]
I tried latest Index of UGS/v2.0.9/ugs-platform-app-ios.dmg
Thanks

**Comment by MikeAghevli** [comment]
Sorry! I think I know what I have done wrong!!!!
Many thanks for quick answer
